src/projections/default-content.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import _ from 'underscore';
+import DefaultContent from '../components/default-content';
+
+function wrap(model) {
+  const { content } = model;
+
+  return _.isString(content) ? {
```

Comment:
People may have `number`, `boolean` content. You may check the other way, if the content is a React element, return it directly.